src/popup/router/components/TransactionList.vue

```diff
@@ -157,10 +158,7 @@ export default {
         this.updateTransactions(transactions);
       } finally {
         this.loading = false;
-      }
-      if (transactions.length) {
         this.page += 1;
-        this.checkLoadMore();
```

Comment:
Without this check we will get in the situation like this, when the chunk contain not enought info to fill the whole height, and the user is not able to scroll to the bottom cause of that
![Ivelin](https://user-images.githubusercontent.com/9008074/126613995-c57a314c-5ed0-4e1f-8814-dbc3d9a5acda.gif)
or even like this:

this PR
![image](https://user-images.githubusercontent.com/9008074/126614236-77e860b3-6c13-40e9-ba85-d048c0bc3be2.png)

develop
![image](https://user-images.githubusercontent.com/9008074/126614319-d8017fc1-ad7c-4448-82d5-39e6b092b01b.png)